presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java

```diff
@@ -265,6 +265,21 @@ public void testOutputStats()
         assertEquals(1L, queryCompletedEvent.getStatistics().getOutputRows());
     }
 
+    @Test
+    public void testGraphvizQueryPlanOutput()
```

Comment:
Instead of having a new test, how about we add an assertion to [testNormalQuery](https://github.com/prestodb/presto/blob/ab6bdcb491a2578f0d432b74997f4df5d8db192e/presto-tests/src/test/java/com/facebook/presto/execution/TestEventListener.java#L158) above and compare against a graphviz plan string that we get from a resource file ?

Reply:
I'd think to keep a separate unit test case because it needs to enable `print_stats_for_non_join_query` session property which I don't think it's related to the normalQuery testcase.

Reply:
Cool - can you add a comment 'EXPLAIN will output graphviz plan with stats. To compare this against this the plan from the event listener graphviz output  `print_stats_for_non_join_query` needs to be set`  (Ignore my comment about using a resource file, I think its better this way - the graphviz node naming scheme can change over time)